docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -73,6 +73,15 @@ Alternatively you can add the properties to the previously created Hadoop config
 </configuration>
 ```
 
+### Check Setup
+
+* You can run the following command in the Alluxio project directory 
+to check if you have successfully setup Spark with Alluxio:
+
+```bash
+$ checker/bin/alluxio-checker.sh Spark <spark master uri> <spark partition number(optional)>
+```
+
```

Comment:
@yupeng9 
I add Spark checker in docs according to the following structure:
## Prerequisites
### General Setup
### Additional Setup for HDFS
### Check Setup
PTAL, Thanks

Reply:
Should I add more explanations for integration checker or just make it easier for users to use it?

Reply:
It would be useful if you can elaborate more on what this checker does. You can take a look at how we wrote about `validateEnv` tool at [here](https://alluxio.org/docs/1.7/en/Getting-Started.html#validating-alluxio-environment)